src/charm.py

```diff
@@ -99,6 +114,17 @@ def _on_get_access_gateway_secrets(self, event: ActionEvent) -> None:
             event.fail(str(e))
             return
 
+    def _on_orchestrator_available(self, event: OrchestratorAvailableEvent):
+        """Triggered when a related orchestrator is made available.
+
+        The AGW will be configured to connect to the orchestrator with the data from
+        the event. Services will then be restarted.
+        """
+        if self._install_configurations(event):
+            self.unit.status = MaintenanceStatus("Restarting Access Gateway to apply changes")
+            self._restart_magma()
+            self.unit.status = ActiveStatus()
```

Comment:
Do you think it makes sense to check whether magma services actually started before going to `ActiveStatus`?

Reply:
Yes, I added it to the latest commit.